cmake/Platform/Windows/PackagingPostBuild.cmake

```diff
@@ -189,7 +189,7 @@ if(CPACK_AUTO_GEN_TAG)
     upload_to_s3(
         ${_latest_upload_url}
         ${_temp_dir}
-        "(${_non_versioned_exe}|build_tag.txt)$"
```

Comment:
Shouldn't the `.` before `txt` be escaped?

Reply:
Technically yes, as would the one in the exe filename too but neither are completely necessary here.  This instance is being run on a temp directory created just before the call, with only those two files, then cleaned up right afterwards.